### PR TITLE
refactor!: changed `KAFKA_PORT` to a `ContainerPort`

### PR DESCRIPTION
**Pull request by CommanderStorm**
refactor!: made sure that `KAFKA_PORT` follow the convention of being a `ContainerPort`

This reverts commit https://github.com/testcontainers/testcontainers-rs-modules-community/commit/1c6ba59f17810316fe72558b8f29054524066927.

It is based on https://github.com/testcontainers/testcontainers-rs-modules-community/pull/183 => if said PR is merged, the unrelated changes (included to make merge-conflicts less likely) will merge/rebase away and only the kafka changes are left.

### CODE REVIEW

**Review by DDtKey — APPROVED** [review]

File: src/kafka/mod.rs
Comment (DDtKey) [inline src/kafka/mod.rs:1]: There is an issue in line 51
```rs
 format!("PLAINTEXT://localhost:{KAFKA_PORT},BROKER://localhost:9092",),
```

It's not u16 used, so it's displayed as `9093/tcp`

File: src/surrealdb/mod.rs [hunk 1]
```diff
@@ -134,7 +134,7 @@ mod tests {
         opt::auth::Root,
         Surreal,
     };
-    use testcontainers::{runners::AsyncRunner, ImageExt};
+    use testcontainers::runners::AsyncRunner;
```
Comment (DDtKey) [inline src/surrealdb/mod.rs:137]: Issue in tests wasn't related to surreldb import, I'd prefer not to touch unrelated modules here
It was a warning, but not causing the failure of test suite. See comment above

Comment (CommanderStorm) [inline src/surrealdb/mod.rs:137]:
```suggestion
    use testcontainers::{runners::AsyncRunner, ImageExt};
```

File: src/kafka/mod.rs
Comment (CommanderStorm) [reply src/kafka/mod.rs:1]: oops

**Pull request merged**